documentation/compatibility/8.0/README.md

```diff
@@ -0,0 +1,79 @@
+
+### Was this documentation helpful? [Share feedback](https://www.research.net/r/DGDQWXH?src=documentation%2Fcompatibility%2F7.0%2FREADME)
+
+# Breaking Changes in 8.0
+
+If you are migrating your usage to `dotnet monitor` 8.0, the following changes might affect you. Changes are grouped together by areas within the tool.
+
+## Changes
+
+| Area | Title | Introduced |
+|--|--|--|
+| Deployment | [The tool will not run on .NET 6 or 7 due to removal of `net 6.0` and `net 7.0` target frameworks](#deployment-only-1-tfm-for-dotnet-monitor); **Note**: The tool will still be able to monitor applications running these .NET versions. | GA |
+| Docker | [Non-Root by default](#docker-non-root-by-default) | GA |
+| Metrics | [Live metrics respects `Metrics` configuration](#metrics-live-metrics-respects-metrics-configuration) | GA |
+
+## Details
+
+### Deployment: Only 1 TFM for dotnet-monitor
+
+The `dotnet monitor` now targets a single framework. See [PR 5501](https://github.com/dotnet/dotnet-monitor/pull/5501). This has no impact on the dotnet-monitor docker image, but installation through `dotnet tool install` will require the dotnet 8 sdk installed.
+
+### Docker: Non-root by default
+
+In `dotnet monitor` 8, dotnet-monitor runs as a non-root distroless container by default. If the application is based on a distroless dotnet 8 aspnet image, no changes are needed. However, if the app is running as root, it may require one of the following mitigations:
```

Comment:
```suggestion
In `dotnet monitor` 8, dotnet-monitor runs as a non-root distroless container by default. If the application is based on a distroless dotnet 8 ASP.NET image, no changes are needed. However, if the app is running as root, it may require one of the following mitigations:
```

nit: We use `ASP.NET` elsewhere in the docs (this'll also resolve the spelling check. Distroless should probably get added to `cspell.json`